raiden/network/proxies/token_network_registry.py

```diff
@@ -132,6 +133,14 @@ def add_token(
                 raise BrokenPreconditionError(
                     "The token is already registered in the TokenNetworkRegistry."
                 )
+            if deprecation_executor == NULL_ADDRESS_BYTES:
```

Comment:
In theory, this could **never** happen because the contracts automatically set this to `msg.sender` in the constructor. 

I think this is checking the wrong thing. Let's first answer the question of: When the constructor of the contract is called, what would this `deprecation_executor` do to actually deprecate the contract? If we have a modifier check in the `createERC20TokenNetwork` function to see if the `deprecation_executor` actually set `is_deprecated == True` then this is something we have to add to the proxies. Otherwise, this change doesn't make sense.